credential-free/spring-cloud-azure-starter-jdbc-mysql/spring-cloud-azure-mysql-sample/terraform/set_up_azure_ad_user.sh

```diff
@@ -0,0 +1,40 @@
+echo "Prepare the working environment..."
```

Comment:
Is it possible to finish this logic by terraform?

Reply:
Here should be to `set up environment variables`, I did not find a way to use terraform to set environment variables. If so, you may need two `main.tf` files to work in `bash` and `Powershell`.

MySQL – Flexible Server does not currently support setting up Azure ad admin user with terraform. hashicorp/terraform-provider-azurerm#19272